runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargeConfig.java

```diff
@@ -183,11 +183,33 @@ default boolean showBellowCharges()
 		return true;
 	}
 
+	@ConfigItem(
+			keyName = "showBasketCharges",
+			name = "Show Basket Charges",
+			description = "Configures if fruit basket item charge is shown",
+			position = 13
+	)
+	default boolean showBasketCharges()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+			keyName = "showSackCharges",
+			name = "Show Sack Charges",
+			description = "Configures if sack item charge is shown",
+			position = 14
```

Comment:
Fix your indentation

Reply:
And in rest of the file as well

Reply:
Are there default settings for Runelite I can import? My settings are always messing up the indentation.

Reply:
you can download our formatter file and setup your imports correctly from here

https://github.com/runelite/runelite/wiki/Code-Conventions